CMakeLists.txt

```
@@ -168,6 +168,10 @@ add_subdirectory(src)
 # utils, programs, examples and tests
 #
 
+if (NOT LLAMA_BUILD_EXAMPLES OR NOT LLAMA_BUILD_COMMON)
```

```suggestion
if (NOT LLAMA_BUILD_EXAMPLES AND NOT LLAMA_BUILD_COMMON)
```

Also add a CMake log message when this branch is executed.

---

On second thought, I think this condition should be just `if NOT LLAMA_BUILD_COMMON`, because we cannot build (most of the) examples without libcommon

---

Fixed in https://github.com/ggml-org/llama.cpp/pull/12761/commits/533e0dcb887b2ee106226084b78460a512788112